app/assets/stylesheets/application.css.erb

```diff
@@ -0,0 +1,251 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
```

Comment:
Feel free to remove Rails comments from files, especially once you start adding your own stuff. It can help keep things clean.